ports/nrf/drivers/bluetooth/download_ble_stack.sh

```diff
@@ -35,6 +35,22 @@ function download_s132_nrf52_5_0_0
     cd -
 }
 
+function download_s132_nrf52_6_1_0
```

Comment:
Not that important, but all these download functions could be refactored into one that takes a few arguments.

Reply:
Maybe next time.

Reply:
yeah, I also feel that. I am not a fan of shell script as well. It is better to have some kind of dictionary for SD name and link :+1:

Reply:
You could rewrite it as a Python script. :smiley:

Reply:
That will be a very awful python script. I am way too bad with python :(